city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,129 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiNorthernIlUniversitySpider(CityScrapersSpider):
+    name = "chi_northern_il_university"
+    agency = "Northeastern Illinois University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.neiu.edu/about/board-of-trustees/board-meeting-materials"]
+
+    def parse(self, response):
+        self.link_date_map = defaultdict(list)
+        link_list = response.xpath(
+            '//div[@class="field field--name-field-generic-body field--type-text-long \
+field--label-hidden field--item"]/h3'
+        )
+        for title in link_list:
+            full_str = title.xpath(".//text()").extract()[0]
+            date = full_str.split("-")[0].split("(")[0]
+
+            date_obj = datetime.strptime(date.strip(" "), '%B %d, %Y')
+            new_date_obj = datetime(date_obj.year, date_obj.month, date_obj.day, 13, 00)
```

Comment:
We don't know the time of the meeting, so the `strptime` default of midnight is fine and we shouldn't reassign the time

Reply:
> For regular board meetings, and unless noted otherwise, Board meetings begin at 1 p.m. and are held at Northeastern Illinois University, 5500 North St. Louis Avenue, Chicago, Ill., 60625. Confirmed meeting locations and times may be found on the agendas for posted meetings on the Board Meeting Materials page.

In fact, regular meeting time is displayed in the website

Reply:
Gotcha, I missed that. It looks like that only applies to board meetings though, so we should only include it there